Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.js

```diff
@@ -159,6 +164,43 @@ WI.ConsoleMessageView = class ConsoleMessageView extends WI.Object
         this._repeatCountElement.textContent = Number.abbreviate(count);
     }
 
+    set timestamp(timestamp)
+    {
+        this._message._timestamp = timestamp;
+        if (this._element)
+            this._renderTimestamp(true);
+    }
+
+    _renderTimestamp(renderlatestTimestamp = false)
+    {
+        if (!this._message._timestamp) {
```

Comment:
This should use the public getter for timestamp (e.g. `this._message.timestamp`)